docs/parameters.md

```diff
@@ -0,0 +1,187 @@
+# Passing parameters to brokers
+
+Table of Contents
+- [Overview](#overview)
+- [Design](#design)
+  - [Basic example](#basic-example)
+  - [Passing parameters as an inline JSON](#passing-parameters-as-an-inline-json)
+  - [Referencing sensitive data stored in secrets](#referencing-sensitive-data-stored-in-secret)
+
+## Overview
+`parameters` and `parametersFrom` properties of `Instance` and `Broker` resources 
```

Comment:
Due to the resource rename, all the references to `Instance`, `Broker`, and `Binding` should be updated to `ServiceInstance`, `ServiceBroker`, and `ServiceInstanceCredential`.

Reply:
@kibbles-n-bytes updated naming, please check.